static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js

```diff
@@ -10,12 +10,31 @@ const shouldRenderLabel = adSlotNode =>
         adSlotNode.getElementsByClassName('ad-slot__label').length
     );
 
+const createAdCloseDiv = (): HTMLDivElement => {
+    const closeDiv: HTMLDivElement = document.createElement('div');
+    closeDiv.className = 'ad-slot__closer';
+    closeDiv.innerHTML =
+        '<svg viewbox="0 0 40 40"><path class="ad-slot__closer__x" d="M 10,10 L 30,30 M 30,10 L 10,30" /></svg>';
```

Comment:
We actually have a number of standard Guardian `.svg`s that can be imported into your `.js` located here https://github.com/guardian/frontend/tree/master/static/src/inline-svgs/icon

Have you looked into using an Icon from here? I'd recommend this approach, they can be used in the following way:

`import crossIcon from 'svgs/icon/cross.svg';`

`closeDiv.innerHTML = `${crossIcon.markup}`;

Reply:
👍